vtex/omnishipping
# CHK-101: Add clearer message for unavailable delivery items

#### What is the purpose of this pull request?

Clarify to users which items are unavailable.

#### How should this be manually tested?

For all examples you need to proceed to Checkout to see the Omnishipping (on shipping step).
You can use any zip code you prefer, e.g.: `20740-580`

[*One item unavailable, pickup available* (pague menos)](https://unavailable--paguemenos.myvtex.com/checkout/cart/add/?sku=44953&qty=1&seller=1&sc=1&sku=54692&qty=1&seller=1&sc=1&sku=27451&qty=1&seller=1&sc=1&sku=50880&qty=1&seller=1&sc=1&sku=53017&qty=1&seller=1&sc=1&sku=6441&qty=1&seller=1&sc=1&sku=56456&qty=1&seller=1&sc=1&sku=52291&qty=1&seller=1&sc=1)
[*One item unavailable, pickup unavailable* (motorola br)](https://unavailable--brmotorola.myvtex.com/checkout/cart/add/?sku=133&qty=1&seller=1&sc=1&sku=10000372&qty=1&seller=1&sc=1&sku=93&qty=1&seller=1&sc=1)

<img width="490" alt="omn_one_unavailable" src="https://user-images.githubusercontent.com/850280/85861108-30355380-b796-11ea-81e1-27e9a1dd7c1e.png">

[*Some unavailable*](https://unavailable2--vtexgame1.vtexcommercestable.com.br/checkout/cart/add?sku=31&qty=1&seller=1&sc=1&sku=334&qty=1&seller=1&sc=1&sku=285&qty=1&seller=1&sc=1&sku=308&qty=1&seller=1&sc=1&sku=291&qty=1&seller=1&sc=1&sku=296&qty=1&seller=1&sc=1&sku=307&qty=1&seller=1&sc=1&sku=331&qty=1&seller=1&sc=1&sku=289&qty=1&seller=1&sc=1&sku=286&qty=1&seller=1&sc=1)

<img width="342" alt="omn_some_unavailable" src="https://user-images.githubusercontent.com/850280/85860991-fc5a2e00-b795-11ea-8d1c-98edb20a0a84.png">


[*All items unavailable*](https://unavailable2--vtexgame1.vtexcommercestable.com.br/checkout/cart/add/?sku=285&qty=1&seller=1&sc=1&sku=308&qty=1&seller=1&sc=1&sku=296&qty=1&seller=1&sc=1&sku=307&qty=1&seller=1&sc=1&sku=331&qty=1&seller=1&sc=1&sku=286&qty=1&seller=1&sc=1)

> If you accessed the "Some unavailable" link, clear your cart before accessing this one (they are the same workspace)

<img width="381" alt="omn_all_unavailable" src="https://user-images.githubusercontent.com/850280/85860886-d6348e00-b795-11ea-8f08-cebe74c9aa62.png">

#### Screenshots or example usage

https://www.figma.com/file/GfG2o8GIgyCWMjNcpmjyrh/04.-Checkout-v6---Shipping---Split?node-id=14%3A2347

#### Types of changes
